src/main/java/gov/cabinetoffice/gapfindapiadmin/service/ApiKeyService.java

```diff
@@ -0,0 +1,21 @@
+package gov.cabinetoffice.gapfindapiadmin.service;
+
+import gov.cabinetoffice.gapfindapiadmin.model.ApiKey;
+import gov.cabinetoffice.gapfindapiadmin.repository.ApiKeyRepository;
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Service;
+
+import java.util.List;
+
+@Service
+@RequiredArgsConstructor
+public class ApiKeyService {
+
+    private final ApiKeyRepository apiKeyRepository;
+
+    public List<ApiKey> getApiKeysForFundingOrganisation(Integer fundingOrgId){
```

Comment:
If you make this an `int` then there's no chance it can ever throw a `NullPointerException`